snuba/migrations/snuba_migrations/events/0007_groupedmessages.py

```diff
@@ -30,11 +30,11 @@ class Migration(migration.ClickhouseNodeMigration):
     def forwards_local(self) -> Sequence[operations.SqlOperation]:
         return [
             operations.CreateTable(
-                storage_set=StorageSetKey.EVENTS,
+                storage_set=StorageSetKey.CDC,
```

Comment:
Same question as above on backwards compatibility. What support do we provide today to users who could have separated their storage sets? I remember we had a discussion about this when designing the cluster abstraction but I cannot remember the outcome.